doc/CLI-user-manual/README.md

```diff
@@ -1144,24 +1203,27 @@ $ kc resolve-did did:mdip:z3v8AuadZVYKXq9oyoWmCgqGREsvMxCKDWxwLHNw3tHpfDyrNr3
 
 ### Encrypting a message
 
-The MDIP CLI can be used to encrypt a message that can only be decrypted by the targeted DID.
-```
+The MDIP CLI can be used to encrypt a message that can only be decrypted by the targeted DID:
+
+```sh
 $ kc encrypt-msg 'this is another secret message' david
 did:mdip:z3v8AuacNPvBNSN8o1LgJxSD9jZVQBkre8BfHdrPgSugb7zuhqs
 ```
 
 ### Decrypting a message or a file
 
-Recipients of MDIP encrypted messages or files can use the command below to decrypt the received content.
-```
+Recipients of MDIP encrypted messages or files can use the command below to decrypt the received content:
+
+```sh
 $ kc decrypt-did did:mdip:z3v8AuacNPvBNSN8o1LgJxSD9jZVQBkre8BfHdrPgSugb7zuhqs
 this is another secret message
 ```
 
 ### Decrypting JSON from a VC
 
-All other MDIP documents are also encrypted. The command below is used to decrypt content from VCs and other documents.
-```
+All other MDIP documents are also encrypted. The command below is used to decrypt content from VCs and other documents:
```

Comment:
```suggestion
Some MDIP documents such as VCs are encrypted JSON. This command command combines decrypting with parsing as JSON:
```

Reply:
Edited your suggestion for a repeated word, hope that's kosher :smile: